README.md

```
@@ -8,6 +8,22 @@
 </div>
 
+## Components
+
+| Component | Description | Image Size (linux/amd64) |
+|-----------|-------------|------------------------|
+| **Agent** | Lightweight Go service that parses Traefik logs and exposes metrics via REST API | <!-- AGENT_IMAGE_SIZE_START -->15MB<!-- AGENT_IMAGE_SIZE_END --> |
+| **Dashboard** | Vite web UI with real-time analytics, charts, and geographic visualization | <!-- DASHBOARD_IMAGE_SIZE_START -->151MB<!-- DASHBOARD_IMAGE_SIZE_END --> |
+| **CLI** | Terminal-based dashboard using Bubble Tea (optional) | N/A |
+
+---
+### Crowdsec-Manager mobile app.
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)